### PR TITLE
22 change db

### PR DESCRIPTION
**Pull request by dawnwages**
Change db and update settings for production.

- Changed collect files output directory name from `static` => `staticfiles` so you'll need to run `collectstatic` and delete the `static` folder locally.

### CODE REVIEW

File: indymeet/settings/dev.py [hunk 1]
```diff
@@ -11,6 +11,13 @@
         }
     }
 
+print('----------------------------------')
+print('----------------------------------')
+print('DEV')
+print('----------------------------------')
+print('----------------------------------')
+
+
```
Comment (RachellCalhoun) [inline indymeet/settings/dev.py:20]: @dawnwages Was this intentionally left in?

**Review by RachellCalhoun — APPROVED** [review]
I left some comments for what appear to be some forgotten about print statements but other than that looks good to me.

File: indymeet/settings/dev.py (hunk 1 above)
Comment (dawnwages) [reply indymeet/settings/dev.py:20]: Yea it was but I can take them out if you think they're ugly.

Comment (RachellCalhoun) [reply indymeet/settings/dev.py:20]: Nope, you are good. Keep it in, i just thought you might have forgotten. :+1:

**Pull request merged**